modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/LifecycleAwareConfigurationInstance.java

```diff
@@ -72,6 +73,7 @@
  */
 public final class LifecycleAwareConfigurationInstance extends AbstractInterceptable implements ConfigurationInstance {
 
+  private static final int ASYNC_TEST_CONNECTIVITY_TIMEOUT = 5;
```

Comment:
this should be configurable, as a sysprop at the very least. 10 seconds is actually too few in some cases.